### PR TITLE
Fix usage example in README.md

### PR DESCRIPTION
**Pull request by yaronius**
Now the code example in README is not working, updated it to be runnable. This closes https://github.com/NicklasWallgren/bankid/issues/7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,41 +26,48 @@ We support the two major Go versions, which are 1.14 and 1.15 at the moment.
...
-}
+func main() {
+	certificate, err := ioutil.ReadFile("path/to/environment.p12")
+	if err != nil {
+		panic(err)
+	}
 
-configuration := configuration.New(
-    configuration.TestEnvironment,
-    &configuration.Pkcs12{Content: certificate), Password: "p12 password"},
```
Comment (yaronius) [inline README.md:45]: here we had an extra brace, removed it

File: README.md [hunk 2]
```diff
@@ -26,41 +26,48 @@ We support the two major Go versions, which are 1.14 and 1.15 at the moment.
...
-if err != nil {
-    panic(err)
-}
+func main() {
+	certificate, err := ioutil.ReadFile("path/to/environment.p12")
+	if err != nil {
+		panic(err)
+	}
 
-configuration := configuration.New(
```
Comment (yaronius) [inline README.md:43]: renamed the var so that it doesn't shadow the package

File: README.md [hunk 3]
```diff
@@ -26,41 +26,48 @@ We support the two major Go versions, which are 1.14 and 1.15 at the moment.
...
-)
+	config := configuration.New(
+		configuration.TestEnvironment,
+		&configuration.Pkcs12{Content: certificate, Password: "p12 password"},
+	)
 
-bankId := bankid.New(configuration)
+	bankId := bankid.New(config)
 
-payload := bankid.SignPayload{PersonalNumber: "<INSERT PERSONAL NUMBER>", EndUserIp: "192.168.1.1", UserVisibleData: "Test"}
```
Comment (yaronius) [inline README.md:50]: fixed the wrong case for payload property

File: README.md [hunk 4]
```diff
@@ -26,41 +26,48 @@ We support the two major Go versions, which are 1.14 and 1.15 at the moment.
...
+	payload := bankid.SignPayload{PersonalNumber: "<INSERT PERSONAL NUMBER>", EndUserIP: "192.168.1.1", UserVisibleData: "Test"}
 
-response, err := bankId.Sign(&payload)
```
Comment (yaronius) [inline README.md:52]: we need context argument for this method

File: README.md [hunk 5]
```diff
@@ -26,41 +26,48 @@ We support the two major Go versions, which are 1.14 and 1.15 at the moment.
...
+		}
 
-    fmt.Printf("%#v", err)
-    return
-}
+		fmt.Printf("%#v", err)
+		return
+	}
 
-fmt.Println(response.Collect())
```
Comment (yaronius) [inline README.md:63]: we need context argument for `Collect()` method too

File: README.md [hunk 6]
```diff
@@ -26,41 +26,48 @@ We support the two major Go versions, which are 1.14 and 1.15 at the moment.
...
 
-payload := bankid.SignPayload{PersonalNumber: "<INSERT PERSONAL NUMBER>", EndUserIp: "192.168.1.1", UserVisibleData: "Test"}
+	payload := bankid.SignPayload{PersonalNumber: "<INSERT PERSONAL NUMBER>", EndUserIP: "192.168.1.1", UserVisibleData: "Test"}
 
-response, err := bankId.Sign(&payload)
+	ctx := context.Background()
+	response, err := bankId.Sign(ctx, &payload)
 
-if err != nil {
-    if response := bankid.UnwrapErrorResponse(err); response != nil {
```
Comment (yaronius) [inline README.md:55]: didn't find the `UnwrapErrorResponse()` function, but it works fine if try to cast it (tested)

**Comment by NicklasWallgren** [comment]
Thanks for the contribution!

**Pull request merged**